src/lib.rs

```diff
@@ -708,6 +708,11 @@ impl<K: KVStore + Sync + Send + 'static> Node<K> {
 		Ok(())
 	}
 
+    /// Checks if [`Node`] runtime is available.
```

Comment:
indentation is a bit off, will fix it.

anything else I should mentioned in the comment? 
because this check is more "codey" check and not really has anything to do with the network, should we mention this?